src/queue/patient-queue.js

```diff
@@ -0,0 +1,57 @@
+class Node {
```

Comment:
Added in https://github.com/healthy-queue/back-end/pull/35 so I think this can also be removed.